tests/integration/api/test_xai.py

```diff
@@ -47,6 +47,11 @@ def test_forward_explain(
     if "dino" in model_name:
         pytest.skip("DINO is not supported.")
 
+    # TODO(someone): fix xai for RTMDet
+    # https://github.com/openvinotoolkit/training_extensions/pull/3433
+    if "rtmdet_inst_tiny" in model_name:
+        pytest.skip("DINO is not supported.")
```

Comment:
```suggestion
        pytest.skip("RTMDet is not supported.")
```
Could you describe skip reason more detail?

Reply:
I'm not sure yet, but I believe it's related to how the data is forwarded. I will take a look in my next PR.

Reply:
For explain functionality please refer detection models. I also added unit tests for this functionality. If explain is enabled in MaksRCNN then https://github.com/openvinotoolkit/training_extensions/blob/dadc5ada3db59128f535c3861524ed625b9a88f6/tests/unit/algo/instance_segmentation/test_maskrcnn.py#L53 this test should be passed

Reply:
At the moment, XAI isn't functioning on both MaskRCNN and RTMDetInst. I believe it would be more efficient to tackle the XAI issue separately in another PR. I'm currently focusing on decoupling data-related aspects, and there's a possibility that some changes might need to be made. Let's address the XAI concerns together once I've completed this phase.

Reply:
Yes sure, I just want to notice that if iseg follow similar logic with detection case, explain feature could be easily enabled